src/components/ThirdPartyCollectionDetailPage/ThirdPartyCollectionDetailPage.tsx

```diff
@@ -294,7 +294,7 @@ export default class ThirdPartyCollectionDetailPage extends React.PureComponent<
                   defaultSelectedLabel={t('third_party_collection_detail_page.synced_filter.all')}
                   defaultValue={this.state.filters.synced}
                   options={[
-                    { value: undefined, text: t('third_party_collection_detail_page.synced_filter.all') },
+                    { value: undefined || '', text: t('third_party_collection_detail_page.synced_filter.all') },
```

Comment:
This fixes the following warning:

![image](https://user-images.githubusercontent.com/8763687/211814901-bb49f48d-e1c3-424d-8bd2-191d3e20ffd5.png)

Reply:
It might have been using the values as key props, right? Great catch!